flux/prow-config.yaml

```diff
@@ -22,6 +22,8 @@ spec:
       domain: "prow.ack.aws.dev"
       presubmitsBucketName: "ack-prow-logs"
       tideStatusReconcilerBucketName: "ack-prow-logs"
+    githubApp:
+      appid:
```

Comment:
Not sure what this is, but this value should be loaded in from a `Secret` in the same way we load the PAT today.

Reply:
Good catch, I had been using it for setting the app ID manually during testing.

Removed.